src/views/OrderDetail.vue

```diff
@@ -97,7 +97,7 @@
 
           <div />
           
-          <ion-checkbox :checked="isParentProductChecked(id)" @ionChange="selectParentProduct(id, $event)" />
+          <ion-checkbox :checked="isParentProductChecked(id)" @click="handleChange(true)" @ionChange="selectParentProduct(id, $event)" />
```

Comment:
Instead of having separate method, I think we could call `selectParentProduct(id, $event)` on click

Reply:
If we call selectParentProduct on click it will not select/deselect all the variants as we won't be able to access the current value of checkbox